README.md

```diff
@@ -514,6 +525,11 @@ kernel and device, changing some options might not be supported.
       tx_usecs_high: 0  # optional mininum=0 maximum=0xffffffff
       tx_usecs_irq: 0  # optional mininum=0 maximum=0xffffffff
       tx_usecs_low: 0  # optional mininum=0 maximum=0xffffffff
+    ring:
+      rx: 0  # optional mininum=0 maximum=0xffffffff
+      rx_jumbo: 0  # optional mininum=0 maximum=0xffffffff
+      rx_mini: 0  # optional mininum=0 maximum=0xffffffff
+      tx: 0  # optional mininum=0 maximum=0xffffffff
```

Comment:
Is this correct? Are we allowing hex values? IMO, `4294967295` would be better.

Reply:
Hex values are allowed.